internal/command/httpserver.go

```diff
@@ -26,7 +26,7 @@ func newHTTPServerRunner(options *httpserver.Options, logger *zap.Logger) *cobra
 		},
 	}
 
-	r.cmd.RunE = func(_ *cobra.Command, args []string) error {
+	r.cmd.RunE = func(_ *cobra.Command, _ []string) error {
```

Comment:
Just FYI, this could be `r.cmd.RunE = func(*cobra.Command, []string) error {`. No need for change.